book/chapters/chapter3/evaluation_and_benchmarking.qmd

```diff
@@ -345,23 +345,23 @@ In the example below, we use a smaller version of `tsk("penguins")` and instanti
 tsk_small = tsk("penguins")$filter(c(1, 100, 200, 300))
 rsmp_customcv = rsmp("custom_cv")
 folds = as.factor(c(1, 2, 1, 2))
-rsmp_customcv$instantiate(tsk_small, f = as.factor(folds))
+rsmp_customcv$instantiate(tsk_small, f = folds)
 resample(tsk_small, lrn_rpart, rsmp_customcv)$predictions()
 ```
 
 ### Stratification and Grouping {#sec-strat-group}
 
 {{< include ../../common/_optional.qmd >}}
 
-Using column roles (@sec-row-col-roles), it is possible to group or stratify observations according to a particular feature or the target.
+Using column roles (@sec-row-col-roles), it is possible to group or stratify observations according to a particular column in the data.
 We will look at each of these in turn.
 
 #### `r index('Grouped Resampling')` {.unlisted .unnumbered}
 
 Keeping observations together when the data is split can be useful, and sometimes essential, during resampling -- spatial analysis (@sec-spatiotemporal) is a prominent example of when this is essential, as observations belong to natural groups (e.g., countries).
 When observations belong to groups, we need to ensure all observations of the same group belong to *either* the training set *or* the test set to prevent potential leakage of information between training and testing.
 For example, in a longitudinal study, measurements are taken from the same individual at multiple time points.
-Grouping ensures that the model is tested on data from each *person*, and not each observation, thereby ensuring that data in the training set is not correlated with data in the test set.
+If we don't do this, we might overestimate the model's generalization capability to unseen individuals, because observations of the same individuals might simultaneously be in the train and test set.
```

Comment:
This doesn't make sense if you remove the other line, which says why we use grouping